src/index.ts

```diff
@@ -34,37 +36,38 @@ function createWindow() {
     width: 1680,
     height: 1050,
     webPreferences: {
-      nodeIntegration: true,
-      contextIsolation: false,
+      preload: path.join(__dirname, "electron-preload.js"),
     },
   });
 
-  if(electronIsDev) {
+  // Setup integration point for access to OS Theme
+  // more: https://www.electronjs.org/docs/tutorial/dark-mode
+  ipcMain.handle("dark-mode", () => {
+    return nativeTheme.shouldUseDarkColors;
+  });
```

Comment:
This is the only change. Other changes were caused by Prettier

Reply:
Do we use dark mode as the default mode now?

Reply:
This gets the native theme defined in the user OS (Mac, Linux or Windows). If they have dark theme setup, then QLDB will start in dark mode - otherwise it will start in light mode